master/buildbot/scripts/windows_service.py

```diff
@@ -230,7 +230,7 @@ def SvcDoRun(self):
             self.info("Starting BuildBot in directory '{}'".format(bbdir))
             hstop = self.hWaitStop
 
-            cmd = '{} --spawn {} start --nodaemon {}'.format(self.runner_prefix, hstop, bbdir)
+            cmd = '{} --spawn {} start --nodaemon {}'.format(self.runner_prefix, int(hstop), bbdir)
```

Comment:
I think this int() deserves a comment as it is really not obvious, and people will clean that up with a {:d}. The bug explain why, but it is too many indirection.